### ISSUE TITLE
Example workspaces

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#1316]
@agrogers has suggested in #1163 that there would be example workspaces.

I'm creating example workspaces named "Workspace 1" and "Workspace 2" and more if needed. I could create there multiple windows, but I can't assume there are more than 1 document installed but if there are, we can use them too. 

What kind of window should I create there exactly?

**Comment by agrogers** [comment]
Yeah, it is a bit tricky when you dont have documents loaded. So not too
much that you could do.

I would suggest workspaces that show off sizing of text and colors. I have
a 'Study' workspace were the text is San-serif and small. And then a
reading workspace where the text is serif and quite a bit larger.

Showing the link window would be good... but what to show in it?

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)

**Comment by tuomas2** [comment]
if commentary is installed we can show commentary in a window that has links. then user is assumed to click a link.
We should show 2 windows max because it should be good for small devices too.

**Comment by tuomas2** [comment]
#1163

**Comment by tuomas2** [comment]
I feel this is a bit risky to implement now.  I don't want to create new bugs at this stage and I'm feeling like releasing. 

We can consider later.

**Comment by agrogers** [comment]
Definitely

___________________________
*www.TenOutOfTen.org* <https://www.TenOutOfTen.org>
***@***.***
(+61) 0477 174 193 (Australia)
(+95) 09 250018669 (Myanmar)
